src/VirtoCommerce.PricingModule.Core/Services/IPriceService.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using VirtoCommerce.PricingModule.Core.Model;
+
+namespace VirtoCommerce.PricingModule.Core.Services
+{
+    public interface IPriceService
```

Comment:
IPriceService : ICrudService<Price>
was there any reason to skip this? Then the conversion could be removed in all the code

Reply:
I am not sure I understand the comment, this is a new interface, PriceService class implements this interface along with the CrudService<..>...